docs/current_docs/cloud/572923-get-started.mdx

```diff
@@ -36,58 +36,65 @@ At the end of this step, you will have signed up for Dagger Cloud and obtained a
 
-Browse to the *Organizations Settings* page of the Dagger Cloud dashboard (accessible by clicking your user profile icon in the Dagger Cloud interface). Select your organization and navigate to the *Configuration* tab. Note the Dagger Cloud token carefully.
+To configure local development, set your Dagger Cloud token as an environment variable. For example: `export DAGGER_CLOUD_TOKEN={your token}`
 
-    ![Get token](/img/current_docs/cloud/get-started/get-token.png)
+:::info BEST PRACTICES FOR VISUALIZING LOCAL RUNS IN DAGGER CLOUD
+Dagger Cloud uses metadata from your version control system (VCS), such as git, to populate some information in the user interface. Therefore, you should run your Dagger pipelines within a VCS context, such as a local directory linked to a cloned git repository.
+:::
 
-Once you have a token, the general procedure to connect Dagger Cloud with a CI provider/CI tool is:
 
-- Store the token as a secret with your CI provider/in your CI tool.
-- Add the secret to your CI environment as a variable named `DAGGER_CLOUD_TOKEN`.
-- For *Team* plan subscribers with ephemeral CI runners only: Adjust the Docker configuration to wait longer  before killing the Dagger Engine container, to give it more time to push data to the Dagger Cloud cache
+### In CI
+The general procedure to connect Dagger Cloud with a CI provider/CI tool is:
 
-:::danger
-You must store the Dagger Cloud token as a secret (not plaintext) with your CI provider and reference it in your CI’s workflow. Using a secret is recommended to protect your Dagger Cloud account from being used by forks of your project.
-:::
+- Add your Dagger Cloud token to your CI workflows.
+  - Store the Dagger Cloud token as a secret with your CI.
+    :::danger Keep your Dagger Cloud token private
+    You must store the Dagger Cloud token as a secret (not plaintext) with your CI provider and reference it in your CI’s workflow. Using a secret is recommended to protect your Dagger Cloud account from being used by forks of your project. We provide links in the steps below for configuring secrets with popular CI tools.
+    :::
+  - Add the secret to your CI environment as a variable named `DAGGER_CLOUD_TOKEN`.
+- For *Team* plan subscribers with ephemeral CI runners only: Add a wait step for the Docker configuration to allow the Dagger Engine to push cache volumes to the experimental Dagger Cloud distributed cache. Subscribers on the *Individual* plan do not have access to the experimental distributed cache and do not need to add this step.
+- If you are using GitHub Actions, install the Dagger Cloud GitHub app for GitHub Checks. This app adds a GitHub Check to your pull requests and links to the run status and pipeline visualization for the commit in Dagger Cloud.
+
+You can use Dagger Cloud whether you're hosting your own CI runners and infrastructure or using hosted/SaaS runners.
 
 :::danger
```

Comment:
Unrelated to this PR, this danger note is a bit confusing. Shall we revisit this one @vikram-dagger ?

Reply:
AFAIR we added it because it was a point of concern for AWS users.